controllers/packagevariants/pkg/controllers/packagevariant/packagevariant_controller.go

```diff
@@ -347,6 +371,29 @@ func (r *PackageVariantReconciler) ensurePackageVariant(ctx context.Context,
 		}
 	}
 
+	// TODO: remove if OK to exclude PackageRevision from cache of r.Client
+	//
+	// var tmpClient client.Client
+	// if tmpClient, err = client.New(config.GetConfigOrDie(), client.Options{
+	// 	Cache: &client.CacheOptions{
+	// 		DisableFor: []client.Object{
+	// 			&porchapi.PackageRevisionResources{}},
+	// 	},
+	// }); err != nil {
+	// 	return nil, err
+	// }
+	var refreshedPR porchapi.PackageRevision
+	if err := r.Client.Get(ctx, types.NamespacedName{Name: newPR.GetName(), Namespace: newPR.GetNamespace()}, &refreshedPR); err != nil {
+		return nil, err
+	}
+	newPR.ResourceVersion = refreshedPR.ResourceVersion
```

Comment:
Why are only ResourceVersion and Tasks are copied and every other potential changes are reverted back?
Why aren't we simply Get() the PR into &newPR, set the condition and update?

Reply:
Done - the copy/revert thing with refreshedPR was left over from solving an earlier hurdle and it isn't needed now.